nowcasting_dataset/data_sources/data_source.py

```diff
@@ -112,15 +112,16 @@ def get_batch(
         Returns: Batch data
 
         """
-        examples = []
+        data = []
```

Comment:
Maybe I'm just being resistent to change(!) :) But, I must admit that, in this function, I feel it would be a tiny bit easier to follow if we renamed `data` to `examples`; and renamed `output` to `example`.  My thinking is that, in the context of this function, an `output` will always be a single training example; and `data` will always be a list of examples.

Reply:
I'm happy to keep it as example